exercises/src/main/scala/fpinscala/errorhandling/Option.scala

```diff
@@ -78,25 +78,4 @@ object Option {
 
   def sequence_with_traverse[A](a: List[Option[A]]): Option[List[A]] = 
     traverse(a)(x => x) 
-}
```

Comment:
set `scroll_past_end = true`
Set to false to disable scrolling past the end of the buffer. If true, Sublime Text will leave a wide, empty margin between the last line and the bottom of the window.

ref: http://sublime-text-unofficial-documentation.readthedocs.org/en/latest/reference/settings.html